README.md

```diff
@@ -98,9 +98,8 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 [images]: ./images
 [kubetest]: https://github.com/kubernetes/test-infra/tree/master/kubetest
 [kubeadm]: https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm/
-[design doc]: ./docs/design/
-[user guide]: ./docs/user/
-[the docs]: ./docs
+[design doc]: ./site/content/docs/design/
+[user guide]: ./site/content/docs/user/
```

Comment:
Also these to the hosted version?
Feel free to leave for a follow up instead. Thanks again.

Reply:
OK, I will update all the doc links to hosted version. :heart: